### PR TITLE
[UI]: Pricing page updates

### LINKED ISSUE #6090: [CI] Add default spreadsheet URL as default value in Feature List workflow
### Desired Situation

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and [designs for Layer5 site](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by SAHU-01**
**Description**

This PR fixes #6090 and adds pricing data info to pricing page, also for auto-deployment feature_data.json path has been modified

image

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 8881409d14f41ee1d88537b25b7d7c84b2048c32 at: https://674f0fb2f5f98d008cc28c2b--layer5.netlify.app

File: src/components/PlanCard/index.js [hunk 1]
```diff
@@ -32,10 +36,14 @@ const PlanCard = ({ planData }) => {
                           ? "0"
                           : x.monthlyprice.toFixed(2)}
```
Comment (leecalcote) [inline src/components/PlanCard/index.js:37]:
```suggestion
                          : x.monthlyprice.toFixed(0)}
```

**Review by leecalcote — COMMENTED** [review]
@SAHU-01 we're in need of a month versus annual pricing switcher.

**Comment by l5io** [comment]
🚀 Preview for commit f3f7bdb78e5c4086534ba880ffbe67775eda84c2 at: https://674f7077d1a3594aaf379937--layer5.netlify.app

File: src/components/PlanCard/planCard.style.js [hunk 2]
```diff
@@ -16,6 +16,84 @@ const PlanCardWrapper = styled.section`
     }
   }
 
+.pricing-container {
```
Comment (leecalcote) [inline src/components/PlanCard/planCard.style.js:19]: Why is all of this duplicated?

**Comment by leecalcote** [comment]
Here we go! Wheeeee!!!

**Pull request merged**